src/App.css

```diff
@@ -1,22 +0,0 @@
-.App {
```

Comment:
Even though it wasn't necessary. It will be useful for the future. Thank you so much! 😄